gulpfile.js

```diff
@@ -449,10 +449,12 @@ function generateBundler (opts, performBundle) {
 
   let bundler = browserify(browserifyOpts)
 
+  process.env.NODE_ENV = opts.devMode ? 'development' : 'production'
```

Comment:
@williamchong007 do we need to set `process.env.NODE_ENV` or just in the bundle?